qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowmodel.py

```diff
@@ -1,170 +1,171 @@
+
+            return DummyInstance()
+
+
+log = Logger("HelpWindowModel")
+
+from qtpy.QtCore import QUrl  # noqa: E402
+from qtpy.QtWebEngineCore import QWebEngineUrlRequestInterceptor, QWebEngineUrlRequestInfo  # noqa: E402
+
+
+def getMantidVersionString():
+    """Placeholder function to get Mantid version (e.g., 'v6.13.0')."""
     try:
         import mantid
 
-        # Use the mantid version object (proper way)
-        versionObj = mantid.version()
-        # Retrieve the patch
-        patch = versionObj.patch.split(".")[0]
-        versionStr = f"v{versionObj.major}.{versionObj.minor}.{patch}"
+        if hasattr(mantid, "__version__"):
+            versionParts = str(mantid.__version__).split(".")
+            if len(versionParts) >= 2:
+                return f"v{versionParts[0]}.{versionParts[1]}.0"
     except ImportError:
-        _logger.warning("Could not determine Mantid version for documentation URL.")
-    except Exception as e:
-        _logger.warning(f"Error determining Mantid version for documentation URL: {e}")
-
-    return versionStr
+        pass
+    log.warning("Could not determine Mantid version for documentation URL.")
+    return None
 
 
 class NoOpRequestInterceptor(QWebEngineUrlRequestInterceptor):
-    """
-    A no-op interceptor that does nothing. Used if we're not loading local docs.
-    """
+    """A no-op interceptor used when loading online docs."""
 
     def interceptRequest(self, info: QWebEngineUrlRequestInfo):
         pass
 
 
 class LocalRequestInterceptor(QWebEngineUrlRequestInterceptor):
-    """
-    Intercepts requests so we can relax the CORS policy for loading MathJax fonts
-    from cdn.jsdelivr.net when using local docs.
-    """
+    """Intercepts requests for local docs (e.g., handle CORS)."""
 
     def interceptRequest(self, info: QWebEngineUrlRequestInfo):
         url = info.requestUrl()
-        if url.host() == "cdn.jsdelivr.net":
+        if url.host() == "cdn.jsdelivr.net":  # Allow MathJax CDN
             info.setHttpHeader(b"Access-Control-Allow-Origin", b"*")
 
 
 class HelpWindowModel:
-    _logger = logging.getLogger(__name__)
-
-    MODE_LOCAL = "Local Docs"
+    MODE_OFFLINE = "Offline Docs"
     MODE_ONLINE = "Online Docs"
 
-    def __init__(self, localDocsBase=None, onlineBase="https://docs.mantidproject.org/"):
-        self._rawLocalDocsBase = localDocsBase
-        self._rawOnlineBase = onlineBase.rstrip("/")
+    def __init__(self, online_base="https://docs.mantidproject.org/"):
+        local_docs_path_from_config = None
+        try:
+            config_service = ConfigService.Instance()
+            raw_path = config_service.getString("docs.html.root", True)
+            if raw_path:
+                local_docs_path_from_config = raw_path
+                log.debug(f"Retrieved 'docs.html.root' from ConfigService: '{local_docs_path_from_config}'")
+            else:
+                log.debug("'docs.html.root' property is empty or not found in ConfigService.")
```

Comment:
`local_docs_path_from_config` isn't set if `raw_path` is empty. This should be an issue with the code further on